bootstrap-sass/pom.xml

```diff
@@ -14,14 +14,12 @@
     <packaging>bundle</packaging>
 
     <properties>
-        <autoModuleName>wicket.bootstrap.saas</autoModuleName>
-
         <antlr.version>3.5.2</antlr.version>
     </properties>
 
     <dependencies>
         <dependency>
-            <groupId>ua.kasta</groupId>
+            <groupId>io.bit3</groupId>
```

Comment:
I think we moved from `bit3` to `kasta` because `bit3` was abandoned and it does not support Mac ARM64.
See https://github.com/l0rdn1kk0n/wicket-bootstrap/pull/986

Reply:
Mainly because of ARM - I believe it continues being abandoned in any case.

Reply:
I'll revert this change

BUT according to https://mvnrepository.com/artifact/ua.kasta/jsass `ua.kasta:jsass` seems to be dead as well :(((
It is "one-release-repo" :((((

Reply:
There is little point in maintaining a java wrapper for a library which is dead. See notice on jsass repo:

> libsass has been [declared deprecated](https://sass-lang.com/blog/libsass-is-deprecated) and there will be no further development! This also ends the further development of jsass. It is not yet clear whether there will ever be an implementation based on the [Sass embedded protocol](https://github.com/sass/embedded-protocol). I recommend switching to [Webpack](https://webpack.js.org/) or similar technologies! This project will continue to be maintained for the time being and compatibility with newer Java versions will be ensured. But please do not expect any further development on my part.

Reply:
fwiw I have switched to using sass with the maven-frontend-plugin some time back.

Reply:
> I have switched to using sass with the maven-frontend-plugin
I did the same :)